### PR TITLE
pangram tests updated to 1.3

### LINKED ISSUE #1018: pangram: update tests to version 1.3.0
As identified in #784, some of the exercise test-suites are out-of-date, `pangram` is one of them. It would therefore be really helpful for people to go through and update the test suite to match the canonical data.

Tests should typically be ordered according to their order in the canonical data, since this makes it easier to check and update them in future. If you think that there's a better order, please let us know your reasoning in your PR.

If there are some additional tests that currently exist in this track but which aren't in the canonical data, these can be kept if you think that they are useful. They should be placed at the bottom of the test cases, and it would also be super-helpful if you could add a comment before them saying `# Additional tests for this track` to make it clear that they aren't part of the canonical data.

The version string should be placed just before the first test class and followed by a blank line. For this exercise, the version string is:
```
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.3.0
```

**Test suite:** [exercism/python/exercises/pangram/pangram_test.py](https://github.com/exercism/python/blob/master/exercises/pangram/pangram_test.py)
**Canonical data:** [exercism/problem-specifications/exercises/pangram/canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json)

### PR DESCRIPTION
**Pull request by luoken**
updated to version 1.3 and moved one test to the bottom

Resolves #1018

### CODE REVIEW

**Comment by m-a-ge** [comment]
@luoken could you please rename your PR to something more meaningful and add a link to the issue in the description

**Comment by cmccandless** [comment]
@luoken Thanks for you pull request! As #1018 is specifically for the `pangram` exercise, please limit your changes in the PR to only those files. If you would like to update `pig-latin` as well, please create a separate PR referencing #1019.

**Comment by luoken** [comment]
link to issue : https://github.com/exercism/python/issues/1018

**Comment by N-Parsons** [comment]
@luoken, you still have changes committed to multiple exercises within this PR. Since the changes for `pig-latin` are already being implemented in #1034, please limit this PR to `pangram`.

File: exercises/pangram/pangram_test.py [hunk 1]
```diff
@@ -42,14 +43,20 @@ def test_missing_letters_replaced_by_numbers(self):
...
-            is_pangram('the quick brown fox jumped over the lazy FOX'),
+            is_pangram('the quick brown fox jumped over the lazy FX'),
+            False)
+        
+    #Additional test for track
+    
+    def test_another_missing_character_x(self):
+        self.assertIs(
+            is_pangram('the quick brown fish jumps over the lazy dog'),
             False)
```
Comment (N-Parsons) [inline exercises/pangram/pangram_test.py]: I don't think that this additional test really adds any coverage?

File: exercises/pangram/pangram_test.py [hunk 2]
```diff
@@ -42,14 +43,20 @@ def test_missing_letters_replaced_by_numbers(self):
 
     def test_pangram_with_mixedcase_and_punctuation(self):
         self.assertIs(
-            is_pangram('"Five quacking Zephyrs jolt my wax bed."'),
+            is_pangram('\"Five quacking Zephyrs jolt my wax bed.\"'),
```
Comment (N-Parsons) [inline exercises/pangram/pangram_test.py]: The `\"` construct is for escaping the quote in the `json` - I don't think that they are needed here.

File: exercises/pangram/pangram_test.py [hunk 3]
```diff
@@ -2,13 +2,16 @@
 
 from pangram import is_pangram
 
-
-# test cases adapted from `x-common//canonical-data.json` @ version: 1.0.0
+# test cases adapted from `x-common//canonical-data.json` @ version: 1.3.0
```
Comment (N-Parsons) [inline exercises/pangram/pangram_test.py]: This should reference `problem-specifications`, ie:

```# Tests adapted from `problem-specifications//canonical-data.json` @ v1.3.0```

**Comment by N-Parsons** [comment]
@luoken, this PR is currently failing Travis-CI check due to `flake8` errors ([see results](https://travis-ci.org/exercism/python/jobs/294136008)). If you're still interested in finishing this PR, these errors need to be fixed, along with a few minor changes that I've pointed out with comments.

**Comment by cmccandless** [comment]
@N-Parsons As this PR abandoned, but nearly finished, I went ahead and made the requested change. If there is nothing else, go ahead and merge.

**Pull request merged**